test/base/src/androidTest/kotlin/io/realm/test/shared/QueryTests.kt

```diff
@@ -669,34 +722,67 @@ class QueryTests {
         realm.writeBlocking {
             // Queries inside a write transaction produce live results which means they can be
             // reused within the closure
-            val countQuery = query(QuerySample::class).count()
+            val countQuery = query<QuerySample>().count()
 
             assertEquals(0, countQuery.find())
             copyToRealm(QuerySample())
             assertEquals(1, countQuery.find())
         }
 
-        realm.query(QuerySample::class)
+        realm.query<QuerySample>()
             .count()
             .find { countValue -> assertEquals(1, countValue) }
     }
 
     @Test
-    @Ignore
     fun count_asFlow() {
```

Comment:
Were these tests added. Don't think I stumbled upon them, but maybe it is because it is opaque when the data set is not visible in the individual test.

Reply:
They were. They can be found in `aggregatorAssertions`. The execution flow is as follows:
1. subscribe to the flow and receive initial aggregated value (0 or null)
2. update the realm
3. receive emission and assert
4. attempt to trick `distinctUntilChanged` by deleting the previously stored objects and save the same ones, resulting into an unchanged aggregate
5. ensure we receive no emissions
6. delete stored values to trigger an emission